lib/drivers/abbottFreeStyle.js

```diff
@@ -429,6 +429,7 @@ module.exports = function (config) {
           .with_timezoneOffset(datum.datetime.offset)
           .with_time(datum.datetime.utc)
           .with_units('mg/dL')
+          .set('index', i)
```

Comment:
Oh yeah I'll add the same comments I've used elsewhere to the effect of 'delete after Jaeb'.